cernopendata/modules/fixtures/data/docs/cms-guide-for-conddb/cms-guide-for-conddb.md

```diff
@@ -6,10 +6,11 @@ These records are stored in the condition database. Condition data include non-e
 
 Most [physics objects](/docs/cms-physics-objects-2011) in the CMS Open Data are already calibrated and ready-to-use, and no additional corrections are needed other than selection and identification criteria, which will be applied in the analysis code. Therefore simple analyses do not need to access the condition database. Examples of such analysis are [the di-muon spectrum example](/record/5001) or [the Higgs analysis example](/record/5500).
 
-However, the access to the condition database is necessary, for example, for jet energy corrections and trigger configuration information. Examples of such analysis are [the PAT object production example](/record/233) or [the top quark par production analysis example](/record/5000).
+However, the access to the condition database is necessary, for example, for jet energy corrections and trigger configuration information. Examples of such analysis are [the PAT object production example](/record/233) or [the top quark pair production analysis example](/record/5000).
```

Comment:
A bit redundant here. Can the second sentence say: `Examples of such analysis are for [the PAT object production](/record/233) or [the top quark pair production](/record/5000).`?